packages/edit-site/src/components/dataviews/view-grid.js

```diff
@@ -1,3 +1,71 @@
-export function ViewGrid() {
-	return 'Grid';
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalGrid as Grid,
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	Card,
+	CardBody,
+	CardMedia,
+	FlexBlock,
+	Placeholder,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import FieldActions from './field-actions';
+
+export function ViewGrid( { data, fields, view, actions } ) {
+	const mediaField = fields.find(
+		( field ) => field.id === view.layoutConfig.mediaField
+	);
+	const visibleFields = fields.filter(
+		( field ) =>
+			! view.hiddenFields.includes( field.id ) &&
+			field.id !== view.layoutConfig.mediaField
+	);
+	return (
+		<Grid gap={ 6 } columns={ 2 }>
+			{ data.map( ( item, index ) => {
+				return (
+					<Card key={ index }>
+						<CardMedia>
```

Comment:
Right now the proportions of the image don't seem correct. I think we may need to support things like aspect ratio in this component or something like that. Curious how designers would like us to adapt how the images render, also whether it's something the user is able to tweak (like changing the aspect ratio).